department-of-veterans-affairs/va.gov-cms
# VACMS-7019 - Create Promo Banner content type

Create a new content type for promo banners

Relates to #7019 .

## Testing done
Reviewed new content type and saw that it fulfilled ACs from parent ticket

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/143299173-f5fc29df-3a88-4e55-8eef-2f5315f8f6f7.png)


## QA steps

As a cms editor
- [x] 1. Visit Full Width Alert (/node/add/banner) and confirm that the name of the content type has been changed to 'Full Width Alert ' from "Banner".
- [x] 2. Visit promo banner creation screen (node/add/promo_banner).
- [x] 3. Confirm the promo type defaults to 'announcement'.
- [x] 4. Confirm that the promo text field is required and has a character limit of 55 characters.
- [x] 5. Confirm that the url field is required and can link to internal or external urls.
- [?] 6. Confirm it utilizes same paths field and workflow as Full Width Alert 
6. Then validate Acceptance Criteria from issue
- [ ]  Content model designed and implemented
-- [ ]  Sharing as many patterns as possible with Banner content type
- [x]  Consider renaming "Banner" to "Full width banner"
- [ ]  Content model documentation has been created, ideally with a screenshot of the design and some annotation for field names. (possible candidate for some Confluence POC for content model documentation)

### Definition of Done


### Select Team for PR review

- [] `Platform CMS team`

### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
